server/models/utils/database.js

```diff
@@ -0,0 +1,62 @@
+import fs from "fs";
+import couchbase from "couchbase";
```

Comment:
You can `import { Cluster, N1qlQuery } from "couchbase"` here directly and simplify the code below, avoiding prefixing usages of those classes or requiring them dynamically.